### DISCUSSION TITLE
error with install nuxt

### DISCUSSION THREAD

**Discussion by cyprianwaclaw** [nuxt/nuxt#25439] [Questions]
I'm trying install nuxt3 project and next install any package as pinia, icon or other. When i just finished install i have error like this: 

`Mac-mini-Cyprian:GitHub cyprian$ npx nuxi@latest init project-name

✔ Which package manager would you like to use?
npm
◐ Installing dependencies...                                                                                                       17:45:17

> postinstall
> nuxt prepare

✔ Types generated in .nuxt                                                                                                        17:46:00

added 727 packages, and audited 729 packages in 42s

121 packages are looking for funding
  run `npm fund` for details

7 high severity vulnerabilities

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.
✔ Installation completed.                                                                                                         17:46:00

✔ Initialize git repository?
No
                                                                                                                                   17:47:16
✨ Nuxt project has been created with the v3 template. Next steps:
 › cd project-name                                                                                                                 17:47:16
 › Start development server with npm run dev  `

when I have write npm audit fix --force I return message about changing nuxt verion to 2. 

I install nuxt with npx nuxi@latest init <project-name> i have the latest version npm, node and sudo 

How i can fixed this problem ?

**Comment by angelhdzmultimedia** [comment]
Did you run `npx nuxi upgrade --force` after creating the project as per the docs?
